backend/src/apiserver/model/task.go

```diff
@@ -33,7 +33,7 @@ type Task struct {
 	Name               string           `gorm:"column:Name; default:null"`
 	ParentTaskId       string           `gorm:"column:ParentTaskUUID; default:null"`
 	State              RuntimeState     `gorm:"column:State; default:null;"`
-	StateHistoryString string           `gorm:"column:StateHistory; default:null;"`
+	StateHistoryString string           `gorm:"column:StateHistory; default:null; size:65535;"`
```

Comment:
@gkcalat, this actually feels a bit inaccurate and unreliable that `size:65535` maps to `longtext` in MySQL, because `text` also accomdates the 65535 size limit [1]. 
I opt to doing it this way in favor of much smaller change and less risky, but I feel like we might want to migrate all the columns that we design to be `longtext` to use a more explicit and reliable gorm size or type.

https://dev.mysql.com/doc/refman/8.0/en/storage-requirements.html#data-types-storage-reqs-strings